tools/plugin-release

```diff
@@ -293,6 +293,7 @@ def _do_build(repo, ver):
                         append = False
                         break
                 if append:
+                    if " " in ls_file: ls_file = "\""+ls_file+"\""
```

Comment:
Won't other characters than space cause issues?
Looking at a more "generic" solution, I've found the following SO answer: https://stackoverflow.com/a/35857 - the best ay seems to use https://docs.python.org/3/library/shlex.html#shlex.quote. Can you test that on a Windows with a space and see if issues really gets fixed? They say it's "not guaranteed to be correct" on windows, but it may works.

Reply:
Or maybe can we act on the subprocess call, see that other answer to the same original question on SO: https://stackoverflow.com/a/35858

Reply:
I do not really know how python is bundled in OS distributions, but as the 3.3. version has been released on 2012, I guess the best solution is to use `shlex.quote`.

Reply:
> the best ay seems to use https://docs.python.org/3/library/shlex.html#shlex.quote. Can you test that on a Windows with a space and see if issues really gets fixed? They say it's "not guaranteed to be correct" on windows, but it may works.

"Executing commands quoted by this module on such shells can open up the possibility of a command injection vulnerability."

Seems like a pretty big risk to use it if it could have security issues on Windows.

> Or maybe can we act on the subprocess call, see that other answer to the same original question on SO: https://stackoverflow.com/a/35858

Seems better. I don't know Python at all, but I'll look at it later.

Reply:
Actually, I'm not sure either solve the issue. I'm not trying to sanitize the entire argument, I just need to ensure files with spaces in their name are treated like a single file.

Reply:
I guess using the process is the way to go. Could you give that a quick try?